config/initializers/secrets.rb

```diff
@@ -4,5 +4,8 @@
 require "velum/secrets"
 
 Rails.application.secrets.secret_key_base = Velum::Secrets.read_or_create_secret_key_base(
-  File.join(ENV["VELUM_SECRETS_DIR"], "key_base.json")
+  File.join(
+    (ENV["VELUM_SECRETS_DIR"] || Rails.root.join("tmp")),
+    "key_base.json"
+  )
```

Comment:
I don't like this option, because on https://github.com/kubic-project/velum/pull/126 we already provide a default for environments `development` and `test`. Or we add something similar to the `production` environment, or we remove those two (`development` and `test`) and leave this change. In this case, we would need to change the mounting point of the container for the secrets...